app/controllers/api/v1/helpees_controller.rb

```diff
@@ -5,6 +5,29 @@ def index
     @helpees = Helpee.all
   end
 
+  def rating
+    @order_request = OrderRequest.find_by('order_id = ? AND order_request_status = ?', params[:order_id], OrderRequest.order_request_statuses[:accepted])
+    if !@order_request.nil?
+      @rating = HelpeeRating.new
+      @rating.order_id = @order_request.order.id
+      @rating.qualifier_id = @order_request.order.helpee.id
+      @rating.qualified_id = @order_request.volunteer.id
+      @rating.score = params[:score]
+      @rating.comment = params[:comment]
+      @rating.save!
+      head :ok
+    else
+      head :bad_request
+    end
+  end
+
+  def rating_pending
+    @helpee_id = params[:helpee_id]
```

Comment:
```suggestion
    helpee_id = params[:helpee_id]
```
Creo que no se utiliza en las vistas, así que no tendría que ser una variable de instancia.